.travis.yml

```diff
@@ -81,7 +81,7 @@ jobs:
                 - composer analyse
 
                 - vendor/bin/phpunit
-                - vendor/bin/behat --strict -vvv --no-interaction --tags="~@todo" || vendor/bin/behat --strict -vvv --no-interaction --rerun --tags="~@todo"
+                - vendor/bin/behat --strict -vvv --no-interaction --tags="~@todo && @application"
```

Comment:
Do we want to run application tests only?

Reply:
Good catch, I used it to speed up feedback loop while working on it, gonna revert it now.